packages/lib-core/src/extensions/core-types.ts

```diff
@@ -0,0 +1,18 @@
+export type K8sVerb =
```

Comment:
Similar to my above comment, let's have a consistent way to represent all types instead of spreading them over the source tree.

@andrewballantyne If we identify an overlap / commonality between k8s `lib-core` types vs. k8s `lib-utils` types, we can move them into the `common` package, whose code & types is bundled into all libraries that use it.